Formula/byteman.rb

```diff
@@ -0,0 +1,61 @@
+class Byteman < Formula
+  desc "Java bytecode manipulation tool for testing, monitoring and tracing."
+  homepage "http://byteman.jboss.org/"
+  url "http://downloads.jboss.org/byteman/3.0.5/byteman-download-3.0.5-bin.zip"
+  sha256 "c7b90b63346d4a0f648c55e12c21110fb173fb9ae8471727fd44906591fa8c26"
+
+  bottle :unneeded
+  depends_on :java => "1.6+"
+
+  def install
+    env = Language::Java.java_home_env("1.6+").merge(:BYTEMAN_HOME => libexec)
```

Comment:
Maybe move this down to where it's actually used in the wrapper script generation?